### DISCUSSION TITLE
Ready To Dump GBoard As Soon As Word Suggestions Are Available.

### DISCUSSION THREAD

**Discussion by atikattar1104** [florisboard/florisboard#2060] [General]
I Have Created A Theme Of FlorisBoard, And I Am In Love With It. All I Am Waiting For Is Word Suggestions To Be Implemented. So That I Don't Have To Rely On Closed Source GBoard, And Ready To Throw It Away. Probably, Hacker's Keyboard's Suggestion Implementation Might Help In Word Suggestion Issue. Let's Keep Fingers Crossed.

**Comment by Beach77** [comment]
I want to dump gboard because it's acting really weird. I press on a letter which shows up on word suggestions but not on the place I'm typing so I have to press multiple times for it to work. Makes my head spin.
